dora/structures/models.py

```diff
@@ -197,6 +207,21 @@ class Structure(ModerationMixin, models.Model):
 
     ape = models.CharField(max_length=6, blank=True)
 
+    accesslibre_url = models.URLField(
+        verbose_name="URL accesslibre",
+        blank=True,
+        null=True,
+        validators=[validate_accesslibre_url],
+    )
+    osm_hours = models.CharField(
+        max_length=255, blank=True, null=True, validators=[validate_osm_hours_str]
+    )
+    hours_details = models.CharField(max_length=255, blank=True, null=True)
+    national_labels = models.ManyToManyField(
+        StructureNationalLabel, blank=True, null=True
+    )
+    other_labels = models.CharField(max_length=255, null=True)
```

Comment:
```suggestion
    other_labels = models.CharField(max_length=255, blank=True)
```
Pas de null sur les chaines de caractère en règle générale (https://docs.djangoproject.com/en/4.1/ref/models/fields/#null)